app/views/keywords/_keyword.html.slim

```diff
@@ -1,3 +1,9 @@
 tr
   td = keyword.keyword
   td = keyword.created_at.to_date
+  td.text-center
+    - if keyword.status == 'in_queue'
```

Comment:
As I remember, if we are using `enum` then Active Record creating some method for us, like `status?` or `is_status?`. You can do a check  @gutakk 

so instead of `if keyword.status == 'in_queue'` we will have `if keyword.status?`

Reply:
Fixed in 62e18feb5b5643d1766aa6f3826a73042d7b56cf